Documentation/install.md

```diff
@@ -36,7 +36,7 @@ $ kubectl gadget version
 ```
 $ git clone https://github.com/kinvolk/inspektor-gadget.git
 $ cd inspektor-gadget
-$ make kubectl-gadget-linux-amd64
+$ make build-ig-linux-amd64
```

Comment:
Oh, I didn't realise that the target was user visible here. Maybe the previous name and the way you did it was better, up to you. Sorry for the back and forth.

Reply:
I switched back to kubectl-gadget, renamed some other "build-ig" references  to "kubectl-gadget" and fixed the "install-user" command.